### PR TITLE
Fix Multiple Display different DPI causing weird staring position

### PR DESCRIPTION
**Pull request by taooceros**
https://github.com/microsoft/PowerToys/pull/33784

Tested with @Garulf about the correct position compared to the current release 1.19

### CODE REVIEW

File: Flow.Launcher/MainWindow.xaml.cs [hunk 1]
```diff
@@ -187,6 +188,7 @@
             WindowsInteropHelper.DisableControlBox(this);
             InitProgressbarAnimation();
             InitializePosition();
+            InitializePosition();
```
[the remark this answers is not in the document]
Comment (taooceros) [reply Flow.Launcher/MainWindow.xaml.cs:191]: This is on intention to prevent the dpi issue across multiple displays.

**Comment by Garulf** [comment]
Current Release 1.19.1:

Screenshot 2024-09-21 141430

This fix:

Screenshot 2024-09-21 141737

**Review by Garulf — DISMISSED** [review]

File: Flow.Launcher/MainWindow.xaml.cs [hunk 2]
```diff
@@ -59,6 +59,7 @@ public MainWindow(Settings settings, MainViewModel mainVM)
 
             InitializeComponent();
             InitializePosition();
```
Comment (jjw24) [inline Flow.Launcher/MainWindow.xaml.cs:61]:
```suggestion
            // Initialize call twice to work around multi-display alignment issue- https://github.com/Flow-Launcher/Flow.Launcher/issues/2910
            InitializePosition();
```

File: Flow.Launcher/MainWindow.xaml.cs [hunk 3]
```diff
@@ -187,6 +192,7 @@ private void OnLoaded(object sender, RoutedEventArgs _)
             WindowsInteropHelper.DisableControlBox(this);
             InitProgressbarAnimation();
             InitializePosition();
```
Comment (jjw24) [inline Flow.Launcher/MainWindow.xaml.cs:194]:
```suggestion
            // Initialize call twice to work around multi-display alignment issue- https://github.com/Flow-Launcher/Flow.Launcher/issues/2910
            InitializePosition();
```

File: Flow.Launcher/MainWindow.xaml.cs [hunk 4]
```diff
@@ -290,50 +302,58 @@ private void OnLoaded(object sender, RoutedEventArgs _)
 
         private void InitializePosition()
         {
-            if (_settings.SearchWindowScreen == SearchWindowScreens.RememberLastLaunchLocation)
-            {
-                Top = _settings.WindowTop;
-                Left = _settings.WindowLeft;
-            }
-            else
+            InitializePositionInner();
```
Comment (jjw24) [inline Flow.Launcher/MainWindow.xaml.cs:305]:
```suggestion
            // Initialize call twice to work around multi-display alignment issue- https://github.com/Flow-Launcher/Flow.Launcher/issues/2910
            InitializePositionInner();
```

File: Flow.Launcher/MainWindow.xaml.cs [hunk 5]
```diff
@@ -642,6 +651,7 @@ private void UpdatePosition()
             if (_animating)
                 return;
             InitializePosition();
```
Comment (jjw24) [inline Flow.Launcher/MainWindow.xaml.cs:653]:
```suggestion

            // Initialize call twice to work around multi-display alignment issue- https://github.com/Flow-Launcher/Flow.Launcher/issues/2910
            InitializePosition();
```

**Review by jjw24 — APPROVED** [review]
Thanks guys 👍 👍

**Pull request merged**